util/util.go

```diff
@@ -283,3 +284,12 @@ func ReadLines(reader *bufio.Reader, count int, maxLineSize int) ([][]byte, erro
 	}
 	return lines, nil
 }
+
+// PlanCacheMatchOpts store some property used to fetch plan from plan cache
+type PlanCacheMatchOpts struct {
```

Comment:
Can you create a plancache package in ```util```? It can avoid unavailable cache for code import util package when updating ```type``` code.

Reply:
Updated, PTAL